source/first-steps/system-requirements.rst

```diff
@@ -16,13 +16,8 @@ PostgreSQL
   latest patches and updates.
 
 Oracle
-  Supported versions are **Oracle 10g R2 and higher**. The 3D City
-  Database requires spatial data support provided either through the
-  Oracle *Spatial* or *Locator* extension. It is highly recommended to
-  install available patches to avoid unexpected errors and to benefit from
-  the latest functionality. For Oracle 10g R2, at least patch set
-  10.2.0.4.0 is required for using the KML/COLLADA/glTF export
-  capabilities.
+  Supported versions are **Oracle 19c and higher**, since the previous versions
```

Comment:
Make two sentences?!

Supported versions are **Oracle 19c and higher**. Previous versions of the Oracle Database have reached end-of-life in July 2022 and are therefore no longer supported by the 3DCityDB.

Reply:
done with 981957d886eb3773c95dbd1f31ace9bc5762b6f3